test/object-storage.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var assert = require('assert');
+
+describe('object-storage connector', function() {
+  it('should be set up as loopback-component-storage', function(done) {
+    this.timeout(10000);
+    var app = require('./fixtures/object-storage-app');
+    var booted = false;
+    app.on('booted', function() {
+      if (booted) return;
+      assert('MyObjectStorage' in app.datasources);
+      var ds = app.datasources.MyObjectStorage;
+      assert.equal(ds.name, 'loopback-component-storage');
```

Comment:
Similarly here, the assertion errors don't specify the property name that was being tested.

```js
expect(ds).to.have.property('name', 'loopback-component-storage');
```

The same comment applies to the lines below too.